scalameta/trees/shared/src/main/scala/scala/meta/internal/Scaladoc.scala

```diff
@@ -97,10 +97,10 @@ object Scaladoc {
   /* List blocks */
 
   /** Represents a list item */
-  final case class ListItem(text: Text, nested: Option[ListBlock] = None)
+  final case class ListItem(terms: Seq[Term])
```

Comment:
since every list item starts with text first, i'd keep `text` and replace `nested` with `terms: Seq[Term] = Nil`